app/travel_processor/webapp/manifest.json

```diff
@@ -15,7 +15,7 @@
     },
     "dataSources": {
       "mainService": {
-        "uri": "processor/",
+        "uri": "/processor/",
```

Comment:
leading `/` had to be removed because otherwise the deployed app doesn't work with UI
without leading `/` both local and deployed app do work
with leading `/` only local app works